### ISSUE TITLE
Unable to execute command for SwipeItem with screen reader on Android

### ISSUE THREAD

**Issue by towebo** [dotnet/maui#23478]
### Description

I've added contextual actions to a collection view's items with the following xaml :

                        <CollectionView.ItemTemplate>
                            <DataTemplate x:DataType="Models:DocumentRow">
                                <SwipeView>
                                <SwipeView>
                                    <SwipeView.RightItems>
                                        <SwipeItems>
                                            <SwipeItem 
                                                Text="{x:Static resource:AppResources.SwipeResetItemText}"
                                                BackgroundColor="{AppThemeBinding Light={StaticResource BackgroundColorLight}, Dark={StaticResource BackgroundColorDark}}"
                                                Command="{Binding Source={x:Reference thisPage}, Path=BindingContext.ResetItemCommand}"
                                                CommandParameter="{Binding}"
                                                />
                                        </SwipeItems>
                                    </SwipeView.RightItems>

When running the app on my device with Android 11 I can double tap and hold and then swipe to the left to show the item. I can focus/select the action using the screen reader and it says "double tap to execute" but when double tapping to execute nothing happens.
Doing the same without the screen reader it works as expected.

### Steps to Reproduce

Clone the following repo where you have an example where you can see and test the problem.:
https://github.com/towebo/SwipeItemLacksA11y.git

In the readme you have the following list of steps to reproduce the issue:

1. Compile an push the app to a device.
2. Start Android Accessibility/TalkBack.
3. Start the app.
4. Double tap on a list item and hold for about a second, then swipe left. This is the same as tap and hold when you're not using a screen reader.
5. The SwipeItem should now be visible to the right.
6. Focus the button for the SwipeItem's command that Says "Tap me" by touching it or swipe right or left to navigate to it depending where on the screen you are.
7. The screen reader says "Tap twice to activate" or similar.
8. Double tap to execute the command.

The result is that nothing happens.

If you turn off the screen reader tapping the action button will show an alert.

The dotnet workload list shows the following for Android:
android                    34.0.113/8.0.100       VS 17.11.35017.193

### Link to public reproduction project repository

https://github.com/towebo/SwipeItemLacksA11y.git

### Version with bug

Nightly / CI build (Please specify exact version)

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android, I was *not* able test on other platforms

### Affected platform versions

Android 11

### Did you find any workaround?

No, unfortunately not.

**Comment by RoiChen001** [comment]
I can repro this issue at Android platform on the latest 17.11.0 Preview 2.1 (8.0.61&8.0.40).
